app/models/mixins/authentication_mixin.rb

```diff
@@ -226,7 +255,22 @@ def authentication_check_types(*args)
     # Let the individual classes determine what authentication(s) need to be checked
     types = authentications_to_validate if self.respond_to?(:authentications_to_validate) && types.nil?
     types = [nil] if types.blank?
-    types.to_miq_a.each { |t| authentication_check(t, options) }
+    types.to_miq_a.each do |t|
```

Comment:
won't this have a nil in it if types is blank?  Is that handled?

Incidentally, we should remove this to_miq_a in favor of `Array()` or `Array.wrap()`